client/src/components/ratingsReviews/RatingsAndReviews.jsx

```diff
@@ -72,6 +74,7 @@ class RatingsAndReviews extends React.Component {
     axios.get(`reviews/meta?product_id=${id}`)
       .then(res => {
         console.log('AXIOS GET RATINGS:', res);
+        this.ratingsData = res.data;
```

Comment:
Before (or instead of) storing this data in several places inside your own component, could you use `updateCache` to share it with the entire app? (e.g., `updateCache('ratings', product_id, res.data`)

Reply:
My component is using parts of ratings and reviews dynamically in several places, which is why I need a few different references. 

Cache has been updated and available to all app. I'll try to refactor my references with a future PR